### PR TITLE
created new profile page files with the same finished code

### PR DESCRIPTION
**Pull request by hyeseung1**
## Proof that your changes work (include screenshots and/or for frontend): 
Screen Shot 2023-03-25 at 4 25 20 PM

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have DM'd the group chat that I have made a PR and need reviewers.
- [x] I haved moved the Jira issue to "Under Review"
- [x] I have run `npx eslint filename.js` on all JavaScript files I've changed in this PR and fixed all displayed errors
- [x] I have checked the Chrome console in Developer Tools to make sure there are no significant errors

### CODE REVIEW

File: src/pages/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import './ProfilePage.module.css';
+import styles from './ProfilePage.module.css'
+
+function ProfilePage() {
+    <img className={styles.image} src = 'tommy.png'></img>
```
Comment (thomaspwang) [inline src/pages/ProfilePage/ProfilePage.js]: you didn't add tommy.png! as a result, it doesn't appear. just add back tommy.png into your PR (put it in the "public" folder and import it from there)

**Review by thomaspwang — CHANGES_REQUESTED** [review]
Looks great to me! Just one change :))

File: src/pages/ProfilePage/ProfilePage.js (hunk 1 above)
Comment (hyeseung1) [reply src/pages/ProfilePage/ProfilePage.js]: Screen Shot 2023-03-28 at 10 21 43 PM

I imported it and made sure that pic was in the public folder

File: src/pages/ProfilePage/ProfilePage.js
Comment (thomaspwang) [inline src/pages/ProfilePage/ProfilePage.js:1]: image

code errors

File: src/pages/ProfilePage/ProfilePage.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import './ProfilePage.module.css';
```
Comment (thomaspwang) [inline src/pages/ProfilePage/ProfilePage.js]: unnecessary line since you're already importing ur css as styles

**Review by thomaspwang — CHANGES_REQUESTED** [review]

File: src/pages/ProfilePage/ProfilePage.js (hunk 2 above)
Comment (hyeseung1) [reply src/pages/ProfilePage/ProfilePage.js]: Screen Shot 2023-03-30 at 1 27 38 PM
I deleted extra line

File: src/pages/ProfilePage/ProfilePage.js
Comment (hyeseung1) [reply src/pages/ProfilePage/ProfilePage.js:1]: Screen Shot 2023-03-30 at 1 28 44 PM
made some changes. It should work now.

File: src/pages/index.js
Comment (DylanHopkins21) [inline src/pages/index.js:1]: I don't think you are supposed to include this file in your PR.

**Review by DylanHopkins21 — CHANGES_REQUESTED** [review]
If I am mistaken, feel free to ignore my only comment. Very strong code otherwise!

File: src/pages/index.js
Comment (thomaspwang) [reply src/pages/index.js:1]: ^^ @hyeseung1 agree, you should change it

**Review by thomaspwang — CHANGES_REQUESTED** [review]

File: src/pages/index.js
Comment (hyeseung1) [reply src/pages/index.js:1]: thanks it should be updated

Comment (hyeseung1) [reply src/pages/index.js:1]: Screen Shot 2023-04-09 at 9 50 22 AM
This is what I have on my end and I pushed on my branch again just to make sure

**Review by DylanHopkins21 — APPROVED** [review]
LGTM

**Review by thomaspwang — APPROVED** [review]
Looks great! Nice Stella

**Pull request merged**